docs/im.md

```diff
@@ -0,0 +1,27 @@
+### IM API
+
+RocketChat's Im Messages.
```

Comment:
Should probably be `RocketChat's IM interface.`  (IM is abbreviation for instance message)

Reply:
Ok, updated.